component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -881,7 +890,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 
 								$row = $event;
 								$times = "";
-								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat" || $jevtask == "range")
+								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat")
```

Comment:
Perhaps you should close the other pull request and stick to this one.

what is the purpose of this change?